gulpfile.js

```diff
@@ -816,10 +819,11 @@ function parseExtensionFlags() {
 
 /**
  * Adds `amp-video-service` to the extension set if a component requires it.
+ * @param {!Array<string>} extensionsToBuild
  * @return {boolean}
  */
-function maybeAddVideoService() {
-  if (!argv.extensions.split(',').find(ext => VIDEO_EXTENSIONS.has(ext))) {
+function maybeAddVideoService(extensionsToBuild) {
+  if (!extensionsToBuild.find(ext => VIDEO_EXTENSIONS.has(ext))) {
     return false;
   }
   argv.extensions += ',amp-video-service';
```

Comment:
Shouldn't you be modifying `extensionsToBuild` here?

Reply:
We are not passing `extensionsToBuild` back. Actually if you check our code we calculate `extensionsToBuild` twice.... If you think that's better I'm happy to do another refactor and make `extensionsToBuild` global.